config.yaml.example

```diff
@@ -1,28 +1,46 @@
 ---
+# Abilita (true) o disabilita (false) la modalità debug per il server IdP
 debug: true
+# IdP server - host
 host: 0.0.0.0
+# IdP server - porta
 port: 8088
+# IdP server - secret key
 secret_key: sosecret
-entityid: "http://<host>:<port>"
+# IdP - entity id
+entityid: "http://spid-testenv:8088"
+# Abilita (true) o disabilita (false) la modalità https per l'IdP
 https: false
-https_key_file: "<absolute-path>"
-https_cert_file: "<absolute-path>"
-description: Spid Test IDP
+# Chiave e certificato utilizzati dall'IdP in modalità https (in coppia con il certificato, vedi parametro successivo)
+#https_key_file: "path/to/key"
+#https_cert_file: "path/to/cert"
+# IdP - descrizione
+description: Spid Test IdP
+# IdP - nome
 name: Spid test IdP
+# Endpoint del server IdP, suddivisi per funzionalità e binding
+# nota: utilizzare path relativi
 endpoints:
   single_sign_on_service:
-    http-redirect: "<relative-url>"
-    http-post: "<relative-url>"
+    http-redirect: "/sso/redirect"
+    http-post: "/sso/post"
```

Comment:
Non possiamo usare un unico endpoint sia per POST sia per Redirect, in modo da semplificare la configurazione? es. `/sso`

Reply:
sì, certo. Avevo differenziato per binding in modo da rendere più esplicita lato utente il tipo di operazione, però se può generare confusione utilizziamo un singolo endpoint (eventualmente con path fisso e non customizzabile).

Reply:
Adesso ho reso gli endpoint unici e sono ancora configurabili a livello di path.